src/main/java/me/karakelley/http/RequestBuilder.java

```diff
@@ -0,0 +1,10 @@
+package me.karakelley.http;
+
+import java.util.Map;
+
+public class RequestBuilder {
+
+  public static Request build(String requestMethod, String requestPath, String requestProtocol, Map<String, String> headers, int port) {
```

Comment:
Initially, I thought it would help to isolate the instantiation of `Request`. If for some reason, `Request` has to change, then the `RequestParser` would have to change for more than one reason. But as I type this, I realized I overlooked the fact that the argument list is identical so I'm not quite sure how much isolation there is at this point.

Reply:
Right. Currently if request did change, this builder would change in exactly the same way. In fact, you'll probably run into this when you start handling POST requests, so let's leave it as-is in this PR and you can decide if/how you want it to change when you get to that point.